rpm/busybox.spec

```diff
@@ -215,7 +215,7 @@ Obsoletes: psmisc < 22.13+git1
 # currently the features differ quite a bit
 cp %{SOURCE2} .config
 yes "" | make oldconfig
-%make_build
+%make_build CRYPT_AVAILABLE=n
```

Comment:
This would require libxcrypt-static.

Reply:
And as there is no such thing as libxcrypt-static, wouldn't this be okay?

Reply:
Can you test with https://github.com/sailfishos/libxcrypt/pull/3?

Reply:
It builds just fine with that (skipping `CRYPT_AVAILABLE=n` and adding BR to libxcrypt-static). Interestingly it doesn't seem to affect binary size, which I assumed would be the case if it actually links with libxcrypt version instead of using some busybox code or if there is a difference in functionality. Requires seem to be also the same which is expected. Maybe the only thing it actually affects is that libxcrypt-devel gets installed?

Reply:
> Maybe the only thing it actually affects is that libxcrypt-devel gets installed?

I'm thinking that the only things `BuildRequries: libxcrypt-static` affects is this and it doesn't really use the static library in the end even then. I would just go with `CRYPT_AVAILABLE=n` instead...